CMakeLists.txt

```diff
@@ -212,7 +212,7 @@ elseif (("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU") OR ("${CMAKE_C_COMPILER_ID}" S
 	set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3")
 
 	# enable somewhat modern C++
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
```

Comment:
> The compiler error suggests std::isnan, but q_shared.h needs to be valid C so that codemp can use it, so that would require a #ifdef __cplusplus or something.

This does work as an alternative, although I still prefer the version in this pull request.

https://github.com/smcv/OpenJK/commits/travis-isnan (builds: <https://travis-ci.org/smcv/OpenJK/builds/87767135>)